blocks/hero-horizontal-tabs/hero-horizontal-tabs.js

```diff
@@ -31,14 +31,22 @@ export default function decorate(block) {
   const image = getImage(block);
   const text = getText(block);
   const tabs = createTabs(block, text);
-
+  if (tabs) {
   // move the tab riders in front
-  const wrapper = block.parentElement;
-  const container = wrapper.parentElement;
-  container.insertBefore(wrapper, container.firstElementChild);
-
-  addTabs(tabs, block);
+    const wrapper = block.parentElement;
+    const container = wrapper.parentElement;
+    container.insertBefore(wrapper, container.firstElementChild);
 
+    addTabs(tabs, block);
+  } else {
+    block.firstElementChild.remove();
+    const divs = document.querySelectorAll('div'); // Select all div elements
+    divs.forEach((div) => {
+      if (div.textContent.trim() === 'Contents') {
+        div.remove(); // Remove the div with the exact text content "Contents"
```

Comment:
Same thing about the comment as the previous one, maybe just add a comment above the whole block explaining it removes the "config" divs and not their content as well